script.js

```diff
@@ -21,31 +21,33 @@ async function main() {
   // Load Pyodide along with the required packages to run docutils.
   // The `pygments` package is included to support code blocks with language specifiers
   // (they automatically trigger syntax highlighting)
-  let pyodide = await loadPyodide({ packages: [ "docutils", "pygments" ] });
+  let pyodide = await loadPyodide({ packages: ["docutils", "pygments"] });
   const elapsedTime = ((Date.now() - startTime) / 1000).toFixed(1);
   outputFrame.contentDocument.write(`Ready in ${elapsedTime}s.\n`);
   // This makes the browser favicon stop the loading spinner
   outputFrame.contentDocument.close();
 
   // Trigger rendering whenever the textarea content changes (typing, pasting, etc.)
-  inputTextarea.addEventListener('input', debouncedConvert);
+  inputTextarea.addEventListener("input", debouncedConvert);
 
   return pyodide;
 }
 let pyodideReadyPromise = main();
 
+// Define the timer outside the function
+// so it can be reused (i.e. reset) across function calls.
+let convertTimer;
+
 // This function runs whenever the input text changes,
```

Comment:
Maybe let's add a line above this to make the context clearer:

```suggestion
// Debounced (delayed) auto-conversion function.
// This function runs whenever the input text changes,
```